src/frontend/screens/Onboarding/JoinProjectQrScreen.tsx

```diff
@@ -47,27 +45,17 @@ const m = defineMessages({
   },
 });
 
-//Cancels the QR code screen and resets the navigation stack so the
-//user CANNOT hit the back button to go back to this screen
-const resetAction = StackActions.reset({
-  index: 0, // <-- currect active route from actions array
-  actions: [NavigationActions.navigate({ routeName: "Home" })],
-});
```

Comment:
this was causing app crashes and it's not necessary if you use `navigation.goBack` since that removes the screen from the stack (i.e. impossible to return to screen using the back button)